modules/opensearch/opensearch.module

```diff
@@ -154,10 +154,234 @@ function opensearch_admin_settings($form_state) {
+    '#size' => 25,
+    '#default_value' => $default_settings['filterQuery'],
+  );
+
+  $form['opensearch_autocomplete']['opensearch_search_autocomplete_settings']['sort'] = array(
+    '#type' => 'radios',
+    '#title' => t('Output sort order', array(), array('context' => 'opensearch_search_autocomplete')),
+    '#options' => array('count' => 'count', 'index' => 'index'),
+    '#default_value' => $default_settings['sort'],
+  );
+
+  $form['opensearch_autocomplete']['opensearch_search_autocomplete_settings']['agency'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Agency', array(), array('context' => 'opensearch_search_autocomplete')),
+    '#size' => 26,
+    '#default_value' => $default_settings['agency'],
+  );
+
+  $form['opensearch_autocomplete']['opensearch_search_autocomplete_settings']['profile'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Profile', array(), array('context' => 'opensearch_search_autocomplete')),
+    '#size' => 26,
+    '#default_value' => $default_settings['profile'],
+  );
+  $form['opensearch_autocomplete']['opensearch_search_autocomplete_settings']['maxSuggestions'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Max suggestions', array(), array('context' => 'opensearch_search_autocomplete')),
+    '#size' => 26,
+    '#default_value' => $default_settings['maxSuggestions'],
+  );
+
+  $form['opensearch_autocomplete']['opensearch_search_autocomplete_settings']['maxTime'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Max request time', array(), array('context' => 'opensearch_search_autocomplete')),
+    '#size' => 26,
+    '#default_value' => $default_settings['maxTime'],
+  );
+
+  $form['opensearch_autocomplete']['opensearch_search_autocomplete_settings']['highlight'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Highlight', array(), array('context' => 'opensearch_search_autocomplete')),
+    '#description' => t('If checked, search prefix will be enclosed in highlight prefix and suffix', array(), array('context' => 'opensearch_search_autocomplete')),
+    '#default_value' => $default_settings['highlight'],
+    '#return_value' => 'true',
+  );
+
+  $form['opensearch_autocomplete']['opensearch_search_autocomplete_settings']['highlight.pre'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Highlight prefix', array(), array('context' => 'opensearch_search_autocomplete')),
+    '#size' => 26,
+    '#default_value' => $default_settings['highlight.pre'],
+  );
+
+  $form['opensearch_autocomplete']['opensearch_search_autocomplete_settings']['highlight.post'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Highlight suffix', array(), array('context' => 'opensearch_search_autocomplete')),
+    '#size' => 26,
+    '#default_value' => $default_settings['highlight.post'],
+  );
+
+  $form['opensearch_autocomplete']['opensearch_search_autocomplete_settings']['minimumString'] = array(
```

Comment:
`minimumString` and `maxSuggestions` are not part of the `opensearch` implementation. They are still in `ting_search_autocomplete`. Remember to remove it from other parts of the opensearch module as well.

Reply:
Minimum string still needs to be removed from this module. It is still a part of `ting_search_autocomplete`.